src/python/WMCore/WMBS/MySQL/Workflow/GetDeletedBlocksByWorkflow.py

```diff
@@ -0,0 +1,159 @@
+"""
+_GetDeletedBlocksByWorkflow_
+
+MySQL implementation of Workflows.GetDeletedBlocksByWorkflow
+
+Retrieves a list of workflows and blocks per workflow with counters for deleted blocks,
+"""
+
+
+from WMCore.Database.DBFormatter import DBFormatter
+
+
+class GetDeletedBlocksByWorkflow(DBFormatter):
+    """
+    Retrieves a list of all workflows and the relative deleted blocks lists
+    """
+    sql = """SELECT
+                 dbsbuffer_block.blockname,
```

Comment:
resolving the conversation since we already agreed on the final query structure to be used in the code and this one is left just for debugging purposes